make.bat

```diff
@@ -80,15 +82,13 @@ if %ERRORLEVEL% NEQ 0 (
 	echo V failed to build itself with error %ERRORLEVEL%
 	rd /s /q vc
 	del v_old.exe
-	del v2.pdb
```

Comment:
why is that deleted, does v not produce v2.pdb anymore?

Reply:
No v2.pdb generated.

Reply:
even after `v up` and `v self` ?

Reply:
Yes, there is v2.exe and v3.exe temp files before.